tests/connections/GithubRepoTest.ts

```diff
@@ -0,0 +1,147 @@
+import { BridgeConfigGitHub } from "../../src/Config/Config";
+import { GitHubRepoConnection, GitHubRepoConnectionState } from "../../src/Connections/GithubRepo"
+import { GithubInstance } from "../../src/Github/GithubInstance";
+import { createMessageQueue } from "../../src/MessageQueue";
+import { UserTokenStore } from "../../src/UserTokenStore";
+import { DefaultConfig } from "../../src/Config/Defaults";
+import { AppserviceMock } from "../utils/AppserviceMock";
+import { UserTokenStoreMock } from "../utils/UserTokenStoreMock";
+import { expect } from "chai";
+import { MessageEvent, TextualMessageEventContent } from "matrix-bot-sdk";
+
+const ROOM_ID = "!foo:bar";
+
+const GITHUB_ORG_REPO = {
+	org: "a-fake-org",
+	repo: "a-fake-repo",
+};
+
+const GITHUB_ISSUE = {
+	id: 1234,
+	number: 1234,
+	user: {
+		login: "alice"
+	},
+	html_url: `https://github.com/${GITHUB_ORG_REPO.org}/${GITHUB_ORG_REPO.repo}/issues/1234`,
+	title: "My issue",
+};
+
+const GITHUB_ISSUE_CREATED_PAYLOAD = {
+	action: "opened",
+	issue: GITHUB_ISSUE,
+	repository: {
+		full_name: `${GITHUB_ORG_REPO.org}/${GITHUB_ORG_REPO.repo}`,
+		id: 1234,
+		html_url: `https://github.com/${GITHUB_ORG_REPO.org}/${GITHUB_ORG_REPO.repo}`,
+	}
+// The type is quite complex for a test.
+};
+
+function createConnection(state: Record<string, unknown> = {}) {
+	const mq = createMessageQueue({
+		monolithic: true
+	});
+	mq.subscribe('*');
+	const tokenStore = UserTokenStoreMock.create();
+	const as = AppserviceMock.create();
+	const githubInstance = new GithubInstance("foo", "bar");
+	const connection = new GitHubRepoConnection(
+		ROOM_ID,
+		as,
+		GitHubRepoConnection.validateState({
+			org: "a-fake-org",
+			repo: "a-fake-repo",
+			...state,
+		}),
+		tokenStore,
```

Comment:
```suggestion
		{} as UserTokenStore,
```
This seems to work as well, and we don't need to create (and define) a mock at all :)